app/processing/payment-request/get-completed-payment-request-by-payment-request-id.js

```diff
@@ -6,9 +6,11 @@ const getCompletedPaymentRequestByPaymentRequestId = async (paymentRequestId, tr
     transaction,
     attributes: [
       'paymentRequestId',
+      'agreementNumber',
       'dueDate',
       'marketingYear',
-      'schedule'
+      'schedule',
+      'value'
```

Comment:
Not sure this function works as the `paymentRequestId` is only set if the payment request was received before the settlement.  That's not always going to be the case.

See output from testing.  Some have the id populated, others don't.

![image](https://user-images.githubusercontent.com/42574977/189324991-f37383b9-5952-40ea-9b51-e93398eef91f.png)

I think before this stage, there needs to be a step to try to add the payment request id if it is missing.  

There is a function that does this on receipt of the return message which could be reused, but I think it has a bug in.  It needs to be filtered to only submitted requests and it will just pick the first it finds.  That means the wrong paymentRequestId will be set here.